### DISCUSSION TITLE
Add option to not lower volume on window becoming inactive

### DISCUSSION THREAD

**Discussion by jai-x** [ppy/osu#15143] [Ideas] [answered]
Would be nice to have the option for the game volume not to lower when becoming inactive.

I realise this is probably heavily linked to the discussion in https://github.com/ppy/osu/discussions/13173 W.R.T having more setting toggles which is not desirable but I figure the use case and UX argument is completely different so it was worth opening a discussion.

**Comment by Susko3** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/16479013/137604578-bd26097c-7879-4d4b-ad28-85590c425361.png)

Just set this to 100%.
